### ISSUE TITLE
Empty data when catching UA_ServerStatusDataType

### ISSUE THREAD

**Issue by SvenManier** [open62541/open62541#780]
When reading the node 'ServerStatus' my result is slightly empty even though the reading didn't fail.
Client and server are 0.2.
```
	isGood(Open62541_READ_RESPONSE_BAD, node);
	if (!(readResponse.resultsSize > 0)) {
		throw OpcClientWrapperException(Open62541_READ_RESPONSE_BAD_RESULT_SIZE, node);
	}
	if (!readResponse.results[0].hasValue) {
		throw OpcClientWrapperException(Open62541_READ_RESPONSE_NO_VALUE, node);
	}
	if (!UA_Variant_isScalar(&readResponse.results[0].value)) {
		throw OpcClientWrapperException(Open62541_READ_RESPONSE_NO_SCALAR, node);
	}
	if (readResponse.results[0].value.type != &UA_TYPES[node.getNodeType()]) {
		throw OpcClientWrapperException(Open62541_READ_RESPONCE_WRONG_TYPE, node);
	}
	UA_ServerStatusDataType *serverStatus = (UA_ServerStatusDataType*) readResponse.results[0].value.data;
```
I added a picture of the ServerStatusDataType and the ReadResponse. Sorry for the quality, the window closed every time i tried to make a normal screenshot.
[UA_Issue.pdf](https://github.com/open62541/open62541/files/444155/UA_Issue.pdf)

**Comment by jpfr** [comment]
Hello @SvenManier ,

reading the serverstatus and buildinfo works fine on my box.
Are you working with a recent commit or v0.2-rc1?

The buildinfo structure you show look freshly initialized (zeroed out).
Can you use wireshark to find out if the empty structure is sent over the network or due to some client error?

https://github.com/open62541/open62541/wiki/Writing-Good-Issue-Reports

**Comment by SvenManier** [comment]
Hello,

Im using a recent commit. Here's the wireshark dump

[Server-Capturing.pcapng.zip](https://github.com/open62541/open62541/files/444551/Server-Capturing.pcapng.zip)

**Comment by jpfr** [comment]
Hey @SvenManier ,

solved on the 0.2 branch. We just did not copy over the buildinfo content in the callback.

Best, Julius
